### ISSUE TITLE
TTRSS doesn't fetch ironed feed

### ISSUE THREAD

**Issue by verlen64** [feediron/ttrss_plugin-feediron#187]
I am running latest version of ttrss-docker and feediron plugin. Both are working, but in some strange way don't communicate. 

- Issue 1: Feediron doesn't fetch recipes from github properly. A red error message pops up reading "error Github API message:" Nothing more unfortunately... Thus I ended up copying recipes manually.

- Issue 2: After saving this configuration and recieving good results in testing mode, nothing happens to the feed in ttrss. It is still the same as before installing and configuring feediron. af_readability is disabled.

Do you have any idea, what causes these issues and how to solve them?

ttrss- Error log shows messages like:

```
E_WARNING (2) | plugins.local/feediron/init.php:705 | Trying to access array offset on value of type null 1. plugins.local/feediron/init.php(705): ttrss_error_handler(Trying to access array offset on value of type null, plugins.local/feediron/init.php) 2. classes/pluginhandler.php(15): add() 3. backend.php(144): catchall(add)  Real IP: 5.28.xxx.xxx Forwarded For: 5.28.xxx.xxx Forwarded Protocol: https Remote IP: 192.168.xxx.xx Request URI: /backend.php User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:102.0) Gecko/20100101 Firefox/102.0
```

* Operating System: Synology DSM 7.1.1
* TTRSS version/git version: v22.12
* PHP version: 8.1
* Feediron Configuration: 

```json
{
    "name": "heise.de",
    "url": "heise.de|telepolis.de",
    "stamp": 1599066467,
    "author": "uqs",
    "match": "heise.de|telepolis.de",
    "config": {
        "type": "xpath",
        "xpath": "article",
        "reformat": [
            {
                "type": "regex",
                "pattern": "\/\\.html\/",
                "replace": ".html?seite=all"
            }
        ],
        "modify": [
            {
                "type": "regex",
                "pattern": "\/<.?noscript>\/",
                "replace": ""
            },
            {
                "type": "regex",
                "pattern": "\/<.?a-(lightbox|bilderstrecke)[^>]*?>\/",
                "replace": ""
            }
        ],
        "cleanup": [
            "h1[contains(@class,'a-article-header__title')]",
            "div[contains(@class,'a-article-header__service')]",
            "figure[@class='branding']",
            "a-collapse[contains(@class,'a-box--collapsable')]",
            "a-teaser",
            "a-ad",
            "aside",
            "a-script",
            "a-img",
            "footer"
        ]
    }
}
```

**Comment by dugite-code** [comment]
Do you have any ad-block like pi-hole? I suspect you might have issues resolving dns queries, that error regarding the Github API suggests to me it's just unable to reach Githubs servers as it has no specific errors.

The reason you filters are not working is because the config files are not a 1:1 copy paste from the main config file. You example should look like this:

```
{
    "heise.de|telepolis.de": {
        "type": "xpath",
        "xpath": "article",
        "reformat": [
            {
                "type": "regex",
                "pattern": "\/\\.html\/",
                "replace": ".html?seite=all"
            }
        ],
        "modify": [
            {
                "type": "regex",
                "pattern": "\/<.?noscript>\/",
                "replace": ""
            },
            {
                "type": "regex",
                "pattern": "\/<.?a-(lightbox|bilderstrecke)[^>]*?>\/",
                "replace": ""
            }
        ],
        "cleanup": [
            "h1[contains(@class,'a-article-header__title')]",
            "div[contains(@class,'a-article-header__service')]",
            "figure[@class='branding']",
            "a-collapse[contains(@class,'a-box--collapsable')]",
            "a-teaser",
            "a-ad",
            "aside",
            "a-script",
            "a-img",
            "footer"
        ]
    }
}
```

**Comment by verlen64** [comment]
Thanks a lot for your fast reply, @dugite-code! This solved my second issue in 10 seconds.
The reason was in fact I had omitted those first lines after pasting the script to testing tab as you prescribed in your Readme. But I didn't omit them in the configuration tab, because I assumed them to be just comments.
Strangely even the first issue didn't occur again. Maybe I touched some pull limit of Github the other day. Thus, I suggest to close this issue completely.
